index.bs

```diff
@@ -5401,6 +5405,60 @@ authenticator=] and [=client device=] SHOULD work together to ensure that the ex
 only to the operating system user that created that [=platform credential=].
 
 
+## Username Enumeration ## {#sctn-username-enumeration}
+
+While initiating a [=registration ceremony|registration=] or [=authentication ceremony=], there is a risk that the [=[WRP]=] might leak sensitive
+information about its registered users. For example, if a [=[RP]=] uses e-mail addresses as usernames and a user attempts to
+initiate an [=authentication=] [=ceremony=] for "alex.p.mueller@example.com" and the [=[RP]=] responds with a failure, but then
+successfully initiates an [=authentication ceremony=] for "j.doe@example.com", then the user can conclude that "j.doe@example.com"
+is registered and "alex.p.mueller@example.com" is not. The [=[RP]=] has thus leaked the possibly sensitive information that
+"j.doe@example.com" has an account at this [=[RP]=].
+
+The following is a non-normative, non-exhaustive list of measures the [=[RP]=] may implement to mitigate or prevent information
+leakage due to such an attack:
+
+- For [=registration ceremonies=]:
+
+    - If the [=[RP]=] uses [=[RP]=]-specific usernames as user identifiers:
+
+        - When initiating a [=registration ceremony=], disallow registration of usernames that are syntactically valid e-mail
+            addresses.
+
+            Note: The motivation for this suggestion is that in this case the [=[RP]=] probably has no choice but to fail the
+            [=registration ceremony=] if the user attempts to register a username that is already registered, and an information
+            leak might therefore be unavoidable. By disallowing e-mail addresses as usernames, the impact of the leakage can be
+            mitigated since it will be less likely that a user has the same username at this [=[RP]=] as at other [=[RPS]=].
+
+    - If the [=[RP]=] uses e-mail addresses as user identifiers:
+
+        - When initiating a [=registration ceremony=], interrupt the user interaction after receiving the e-mail address to
+            register and send a message to this address, containing an unpredictable one-time code and instructions for how to use
```

Comment:
nit: 
after receiving the e-mail address to register => after the e-mail address is supplied

..?

Reply:
Nice, thanks!

Reply:
you're welcome :)